### ISSUE TITLE
Issue in running the project

### ISSUE THREAD

**Issue by ghost** [Gottox/socket.io-java-client#125]
Hi I just downloaded this and imported it into eclipse, but when I try to run the basicExample.java I get,

Error while handshaking Server returned HTTP response code: 400 for URL: http://localhost:3000/socket.io/1/

Any suggestion?
1. Not behind proxy.
2. Node server is running on same port 3000(changed in both .js an .java)

**Comment by luisloaiza** [comment]
I was getting at http://host:3000/socket.io/1/ a message of unknown transport. Seems to be a problem of compatibility with the server version of socket io and this client version.

**Comment by ghost** [comment]
Yes after some search i got the issue and got it working but will the project be supporting newer version? any idea?
